### DISCUSSION TITLE
Repeating arm/disarm signals

### DISCUSSION THREAD

**Discussion by xtraorange** [tsightler/ring-mqtt#299] [Support Q&A] [answered]
Hello (and welcome back!),

I'm having an issue where when I send an arm/disarm, it will repeat the action 3 times (ARMED will alert three times from the security system).  I'm not sure where to start in working out why this is occurring, but if you can give me some direction I'd be happy to dig into it myself first.

Any help is appreciated!  Thank you!

**Comment by tsightler** [comment]
It's not an issue I've ever seen, but the logs should provide at least some insight into what is going one.  The code as written makes 5 attempts to set the alarm to the requested mode, with a 10 second delay between each retry.  Basically it attempts to set the alarm, waits 1 second to see if the alarm has entered that state, and, if not, waits 10 seconds and tries again.  Not sure why this would fail as it's pretty basic and been around since the earliest days of this project with little to know changes.

Probably should be an issue vs Q&A

**Comment by xtraorange** [comment, marked as answer]
I only had Feature Requests, General, and Q&A as options.  Would general have been more appropriate?

So, interestingly, with the last update, apparently this issue is resolved.  Not sure why, but glad it's working.  Sorry for the trouble!

**Comment by tsightler** [comment]
No problem, I probably need to add better descriptions for the discussion groups and maybe rename the groups.  I've only just enabled the discussions tabs so I'm still learning how the various group options work myself.  To me the Q&A features seems mostly useful for things that have a question and a single "best" answer (i.e. the answers can be upvoted), i.e. How do I do X.  It seems less useful for things that might require multiple back and forth responses.  I'll think about how to better organize this going forward.

**Comment by tsightler** [comment]
Also, I have no idea why the new version would fix this, there's literally no functional code changes, although there was one extremely minor change to the initialization for the security-panel device, so maybe somehow that's it, but I don't see how this could actually be the case.  Glad it's working though!

**Comment by xtraorange** [comment]
It's possible it was something else entirely.  I hadn't tested it for several days (because of the cycling - annoyed my wife), so maybe something else was interfering or something.  Whatever the case, it works now!
THanks for your help!
